modules/apps/commerce/commerce-product-definitions-web/src/main/resources/META-INF/resources/edit_attachment_file_entry.jspf

```diff
@@ -96,34 +96,32 @@
 	</aui:form>
 </commerce-ui:side-panel-content>
 
-<aui:script>
+<%
+String fieldValues = "[]";
+
+if (cpAttachmentFileEntry != null) {
+	fieldValues = cpAttachmentFileEntry.getJson();
+}
+%>
+
+<aui:script require="commerce-frontend-js/utilities/forms/formsHelper as FormsHelper">
 	var fieldValues = [];
 
+	try {
+		fieldValues = JSON.parse('<%= fieldValues %>');
+	}
+	catch(_ignore) {
+	}
```

Comment:
@Commando18 running `ant format-source-current-branch` on portal-impl should auto fix this.
It should format it to something like this `catch (_ignore) {}`.